web/src/pages/billing/components/BatchGrid.tsx

```diff
@@ -279,7 +280,8 @@ const BatchCard: React.FC<{ item: AnalysisCostRecordBatch }> = ({ item }) => {
                     >
                         Cost by Sequencing Groups -{' '}
                         {item.seq_groups?.filter((s) => !!s.sequencing_group)?.length || 0}{' '}
-                        sequencing group(s)
+                        sequencing group(s) across{' '}
+                        {item.seq_groups?.filter((s) => !!s.stage)?.length || 0} stages
```

Comment:
This is the number of stage / sg combos right, my expectation would be the number of distinct stages across all sequencing groups, eg:

```
SG1 <stage1>
SG2 <stage1>
```
Should be 2 sequencing group(s) across 1 stage(s)

Reply:
Well, just testing it and it is wrong it double count the stages per each seq group.